vm-setup/roles/libvirt/tasks/network_teardown_tasks.yml

```diff
@@ -23,6 +23,12 @@
 
   when:
     - ansible_distribution == 'Ubuntu'
-
   become: yes
 
+- name: Disable the firewall forward rule
+  iptables:
+    chain: FORWARD
```

Comment:
This removes all rules on the forward chain. It is too strong in the way that you would be removing rules that are not added by Metal3-dev-env. You should selectively remove the rules. An alternative approach could be to parameterize the firewall role, so that the state is present when you do make, and the state is absent when you do make clean.

Reply:
Done

Reply:
I tried to find a connection with firewall rules creation during we call the setup playbook and firewall deletion with tear down playbook file. Here during the setup process the firewall rules are generated with separate call in 02_configure_host.sh, it doesn't have have any link with the setup playbook ansiable file.  So if I take this path, need to modify different setting and eventually will be complicated to follow.  That is the reason I take the 2nd path with tear down process where I generate a new file named firewall_teardown_tasks and from here we can delete each firewalls rules generated by metal3-dev-env. The testing works properly when I do `make clean` and `make`command.  After make command all the rules are back in the iptables output..

Reply:
updated